Composer/packages/lib/code-editor/src/utils/common.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export function processSize(size) {
```

Comment:
this looks like a more of a css-in-js concern. can you comment in the code to what this does and why it is needed?

Reply:
applied to `RichEditor.tsx`.